frontend/projects/upgrade/src/app/shared-standalone-component-lib/components/common-section-card-content/common-section-card-content.component.ts

```diff
@@ -17,15 +17,19 @@ import { SharedModule } from '../../../shared/shared.module';
 //
 // Example Usage:
 //
-// contentDetails = [
-//   { key: 'name' },
-//   { description: 'something' },
-//   { tags: ['Tag1', 'Tag2'] },
-//   { Appcontext: 'Context1' },
-// ];
+// contentDetails = {
+//   Key: 'name',
+//   Description: 'something',
+//   Tags: ['Tag1', 'Tag2'],
+//   app_Context: 'Context1',
+// };
 //
 // Simply pass the data to the component as shown below:
 // <app-common-section-card-content [data]="contentDetails"></app-common-section-card-content>
 export class CommonSectionCardContentComponent {
```

Comment:
I just noticed this, I think we want to call this component a `CommonSectionCardOverviewDetailsComponent` instead of `CommonSectionCardContentComponent`, this is specific to overview details, there will be several different components that can be "Section Card Content".